drafts/versions/02/draft-ietf-ippm-ioam-data-02.txt

```diff
@@ -1152,8 +1209,9 @@ Internet-Draft           In-situ OAM Data Fields           February 2018
       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 
 
-   IOAM-E2E-Type:  8-bit identifier of a particular in situ OAM E2E
-      variant.
+   IOAM-E2E-Type:  16-bit identifier of a particular in situ OAM E2E
```

Comment:
A 16-bit identifier which specifies which data types are used in the E2E option data. The IOAM-E2E-Type value is a bit field. The order of packing ...